### PR TITLE
PS-7020: convert MTR tests from python2 to python3

### PR DESCRIPTION
**Pull request by inikep**
1. Fix failing `rocksdb_stress.rocksdb_stress`
2. Convert `python2` scripts to support both `python2` and `python3`

Current requirements to run these scripts are: 
a) python2.6+ (because of `from __future__ import print_function`) 
b) `python` linked to either `python2` or `python3`

### CODE REVIEW

File: mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SET @@session.default_storage_engine = ROCKSDB;
```
Comment (dutow) [inline mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result:1]: how is this related to the python change?

File: mysql-test/suite/rocksdb_stress/t/rocksdb_stress.test [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # basic stress tests for myrocks, just runs the load generator without any crashes
+SET @@session.default_storage_engine = ROCKSDB;
```
Comment (dutow) [inline mysql-test/suite/rocksdb_stress/t/rocksdb_stress.test:2]: how is this related to the python change?

File: mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result (hunk 1 above)
Comment (inikep) [reply mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result:1]: This is required to run this test. It was failing before.

**Review by dutow — APPROVED** [review]

File: mysql-test/suite/rocksdb_stress/t/load_generator.py [hunk 3]
```diff
@@ -312,12 +312,12 @@ def populate_table(num_records):
   if num_records == 0:
     return False
 
-  num_workers = min(10, num_records / 100)
+  num_workers = int(min(10, num_records / 100))
```
Comment (percona-ysorokin) [inline mysql-test/suite/rocksdb_stress/t/load_generator.py]:
```suggestion
  num_workers = min(10, int(num_records / 100))
```

**Review by percona-ysorokin — APPROVED** [review]
LGTM with minor comment
(can be ignored)

**Comment by inikep** [comment]
Tested at https://ps57.cd.percona.com/job/percona-server-5.7-param/272/

**Comment by inikep** [comment]
This PR was merged with https://github.com/percona/percona-server/pull/3794
